js/ui/grid_core/ui.grid_core.header_panel.js

```diff
@@ -116,10 +152,28 @@ const HeaderPanel = ColumnsView.inherit({
     },
 
     optionChanged: function(args) {
+        const parts = getPathParts(args.fullName);
+
```

Comment:
Since "parts" is used only  under the following condition:

```js
if(args.name === 'toolbar')
```
move this variable declaration there.